Hackney.Shared.HousingSearch.Tests/Models/TenureTests.cs

```diff
@@ -27,6 +27,7 @@ public void DomainTenureHasPropertiesSet()
             createdDomainTenure.EndOfTenureDate.Should().BeEquivalentTo(aQueryableTenure.EndOfTenureDate);
             createdDomainTenure.TenureType.Should().BeEquivalentTo(aQueryableTenure.TenureType);
             createdDomainTenure.TempAccommodationInfo.Should().BeEquivalentTo(aQueryableTenure.TempAccommodationInfo);
+            createdDomainTenure.TempAccommodationInfo.AssignedOfficer.Should().BeEquivalentTo(aQueryableTenure.TempAccommodationInfo.AssignedOfficer);
```

Comment:
The above test for `TempAccommodationInfo` should cover the `AssignedOfficer` too,  so this is not strictly necessary.

Reply:
True.. addressed in [b4fab62](https://github.com/LBHackney-IT/housing-search-shared/pull/74/commits/b4fab62b7619bde7c9f9f22115f05198997afcf4)